### PR TITLE
Split out Docker-based value-add stuff to separate repo

### PR DESCRIPTION
**Pull request by peterbourgon**

### CODE REVIEW

**Comment by mrnetops** [comment]
Still finalizing my repos, current work in progress
* https://github.com/mrnetops/fastly-exporter-docker
* https://github.com/mrnetops/fastly-dashboards

I should have final links for you next week (not sure if I'm going to rename the repos before I go final)

**Comment by peterbourgon** [comment]
@mrnetops Cool! I'll wait for your signal. How are things going otherwise?

File: README.md [hunk 1]
```diff
@@ -27,31 +27,41 @@ Avaliable as [mrnetops/fastly-exporter][container] from [Docker Hub][hub].
 docker pull mrnetops/fastly-exporter
 ```
 
+See [TODO][todo] for a complete docker-compose based out-of-the-box experience,
+including pre-built Grafana dashboards.
+
+[todo]: https://example.com
+
```
Comment (peterbourgon) [inline README.md:34]:
```suggestion
See https://github.com/mrnetops/fastly-exporter-docker for a Dockerfile, and
https://github.com/mrnetops/fastly-dashboards for a set of pre-built Grafana
dashboards.
```

**Comment by peterbourgon** [comment]
@mrnetops I'll merge this shortly unless there's an objection. Thanks again for the work!

**Pull request merged**